ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -276,6 +280,10 @@ unsafe fn init(
         slice::from_raw_parts(opts.vslogger_mod_list, opts.vslogger_mod_size as usize)
     };
 
+    if opts.ptrfunc.is_some() {
```

Comment:
No need to check if this is Some or not - we can just set our global regardless.